### DISCUSSION TITLE
[Discussion Thread] Townhome Square from XOHR City 1 - Remake?

### DISCUSSION THREAD

**Discussion by KadTheHunter** [KadTheHunter/CityPlans#13] [General]
**Describe the Idea or Change below:**
Sadly don't have screenshots, but I can go on memory.
Large (very large) square, backs of the houses facing the road (although in the remake we should have other buildings behind them)
4-6 houses, each with a garage, tried to mimic the brownstone style, albeit modded. 
PotatoSheepMilk would remember but who knows where she is, and DRCH16 may remember.
Make it be in the northern districts of the city, as it's a bit more posh/antique.

**Comment by ParadiseMCPE** [comment]
I might be able to contact Sheep but I don't know where you guys stand socially so I won't until you give me the green-light.

**Comment by ParadiseMCPE** [comment]
> I might be able to contact Sheep but I don't know where you guys stand socially so I won't until you give me the green-light.

Her instagram is @sheep.draws so if one of your people has instagram that's an easy way.

**Comment by KadTheHunter** [comment]
> I might be able to contact Sheep but I don't know where you guys stand socially so I won't until you give me the green-light.
> Her instagram is @sheep.draws so if one of your people has instagram that's an easy way.

Thank you for the offer, but it is unnecessary. I resolved that I would not reach out to any former acquaintances, even those I still like such as Sheep.

**Comment by ParadiseMCPE** [comment]
> > I might be able to contact Sheep but I don't know where you guys stand socially so I won't until you give me the green-light.
> > Her instagram is @sheep.draws so if one of your people has instagram that's an easy way.
> 
> Thank you for the offer, but it is unnecessary. I resolved that I would not reach out to any former acquaintances, even those I still like such as Sheep.

sad... she seemed like she'd cut off her ties to the rest in the end but I guess that make sense.

**Comment by KadTheHunter** [comment]
> sad... she seemed like she'd cut off her ties to the rest in the end but I guess that make sense.

I get the same feeling, particularly due to her sudden disappearance and absence around March/April 2020. If she figured out to just move on, then I tip my hat (so to speak) to her for figuring out what took me much longer to figure out.

**Comment by ParadiseMCPE** [comment]
> > sad... she seemed like she'd cut off her ties to the rest in the end but I guess that make sense.
> 
> I get the same feeling, particularly due to her sudden disappearance and absence around March/April 2020. If she figured out to just move on, then I tip my hat (so to speak) to her for figuring out what took me much longer to figure out.

yeah we'd kept in touch for a while after march but I think I was the only one she still spoke to

**Comment by KadTheHunter** [comment]
Looking back on this, TL;DR is *yes*. 
Longer answer is *not yet*. Currently we have nothing even approximating a neighborhood or northern district, so this will be a *long*  way in the future.
